hosts/common.nix

```diff
@@ -48,12 +48,25 @@ in {
       max-free = asGB 200;
       # check the free disk space every 5 seconds
       min-free-check-interval = 5;
+      # Fallback quickly if substituters are not available.
+      connect-timeout = lib.mkDefault 5;
+      # The default at 10 is rarely enough.
+      log-lines = lib.mkDefault 25;
     };
     # Garbage collection
     gc.automatic = true;
     gc.options = pkgs.lib.mkDefault "--delete-older-than 7d";
+
+    daemonCPUSchedPolicy = lib.mkDefault "batch";
+    daemonIOSchedClass = lib.mkDefault "idle";
+    daemonIOSchedPriority = lib.mkDefault 7;
   };
 
+  # Make builds to be more likely killed than important services.
+  # 100 is the default for user slices and 500 is systemd-coredumpd@
+  # We rather want a build to be killed than our precious user sessions as builds can be easily restarted.
+  systemd.services.nix-daemon.serviceConfig.OOMScoreAdjust = lib.mkDefault 250;
```

Comment:
An alternative way of doing this would be something similar to what is currently done on jenkins-controller: https://github.com/tiiuae/ghaf-infra/blob/b0dbfa5e8618930d0bc9014b358187eead71ec99/hosts/azure/jenkins-controller/configuration.nix#L294-L305.

I don't have strong opinnions one way or another, as long as we can make sure this configuration actually works.

Reply:
Looking at https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/system/boot/systemd/oomd.nix, oomd is already enabled, so slight preference over oomd than earlyoom.

Reply:
So, what do you want me to do with it?

Reply:
@tervis-unikie : keep it as it now is in ac267d9a471961b5a9bbd95cf7e5b6baa3204f79

Reply:
:+1: